### PR TITLE
Added Seagrass recipe (suggestion #398 from Mr.Moules)

### PR DESCRIPTION
**Pull request by craftish37**
## Description

Simple add of an item following player suggestion
## Proposed changes

Adding Seagrass organic food, crafted with the Food Fabricator
## Related Issues (if applicable)

--- No Issues related ---
## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

## Anything else
This is a speedly-made PR, I didn't searched that much where this add could break so feel free to notice me 👌
image
Work as intended with "some" add-ons"

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FoodFabricator.java
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FoodFabricator.java:32]: When was sea grass added?

Comment (Sefiraat) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FoodFabricator.java:32]: 1.13

Comment (craftish37) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FoodFabricator.java:32]: In terms of version ? 1.13.x

Comment (Sefiraat) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/FoodFabricator.java:32]: The SlimefunItemStack you reference here `SEAGRASS_ORGANIC_FOOD` doesn't exist. This is making the build fail

**Review by Sefiraat — CHANGES_REQUESTED** [review]

**Comment by Sefiraat** [comment]
Lovely looks like it’s building so that’s good! 

You’ve not ticked that you have tested this change so I can’t approve it until you do. Please build locally and ensure it’s all working as expected if you could :)

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java
Comment (variananora) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java:2390]:
```suggestion
        new OrganicFertilizer(itemGroups.misc, SlimefunItems.SEAGRASS_FERTILIZER, SlimefunItems.SEAGRASS_ORGANIC_FOOD)
```
formatting

**Comment by craftish37** [comment]
Seems pretty good for now, any formatting/ other requests ? 🙂

**Review by variananora — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM

**Review by svr333 — APPROVED** [review]

**Comment by TheBusyBiscuit** [comment]
We need ALL the reviews here apparently :eyes:

**Comment by svr333** [comment]
yeah, this PR is world altering I thought Id give my much needed approval

**Comment by FN-FAL113** [comment]
thats a lot of assignees dope

**Comment by craftish37** [comment]
That’s a lotta people for some watery grass pieces 🤣

**Review by Sefiraat — APPROVED** [review]
Ok, I have 69 required urgent changed before this can go live. Lol jk, LTGM

**Comment by craftish37** [comment]
Needs anything else to merge ?

**Comment by Sefiraat** [comment]
> Needs anything else to merge ?

Nope! Just gotta wait for cookie or walshy to merge now

**Pull request merged**
